src/modelfree/score_agent.py

```diff
@@ -105,9 +149,28 @@ def env_fn(i):
     if record_traj:
         venv.save(save_dir=record_traj_params['save_dir'])
 
+    for env_video_dir in video_dirs:
+        try:
+            for video_file_path in os.listdir(env_video_dir):
+                if 'mp4' in video_file_path:
+                    env_number = env_video_dir.split("/")[-1]
+                    print("Env number: {}".format(env_number))
```

Comment:
Remove or turn into a logger.debug statement

Reply:
Ah, yep, that should have just gotten removed